README.md

```diff
@@ -20,3 +20,4 @@
 # Run `pre-commit` locally.
 
 `pre-commit run --all-files`
```

Comment:
README generally consists of description about the software and the how to operate it. Please update.